src/main/java/core/basesyntax/ShapeSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ShapeSupplier {
+    private final int defaultSize = 20;
+    private final int maxSize = 100;
+    private final int maxBound = 10;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random randomizer = new Random();
+
+    private int randomSize() {
+        return 1 + randomizer.nextInt(maxSize);
+    }
+
+    public Shape getRandomShape() {
+        int index = randomizer.nextInt(maxBound);
+        switch (index) {
+            case 0:
+                return new Circle(colorSupplier.getRandomColor(), randomSize());
+            case 1:
+                return new IsoscelesTrapezoid(colorSupplier.getRandomColor(),
+                        randomSize(), randomSize(), randomSize());
+            case 2:
+                return new Rectangle(colorSupplier.getRandomColor(),
+                        randomSize(), randomSize());
+            case 3:
+                return new RightTriangle(colorSupplier.getRandomColor(),
+                        randomSize(), randomSize());
+            default:
+                return new Square(colorSupplier.getRandomColor(), randomSize());
+        }
+    }
+
+    public Shape getDefaultShape() {
+        return new Circle(String.valueOf(Colors.WHITE), defaultSize);
```

Comment:
The `getDefaultShape` method should return a white circle with a radius of 10, as specified in the task description. Currently, it returns a circle with a size of 20. This needs to be corrected to meet the task requirements.